src/bindings/python/tests/test_runtime/test_model.py

```diff
@@ -146,6 +146,28 @@ def test_get_result_index_invalid():
     assert model.get_result_index(invalid_output) == -1
 
 
+def test_get_result_index_with_result():
```

Comment:
These two tests and `test_parameter_index():` may be joined using `@pytest.parametrize` like you did in `reshape` PR :)